schema.py

```diff
@@ -2,7 +2,149 @@
 from graphene.types.datetime import Date, Time
 
 class Data(object):
-  dining_halls = {}
+  eateries = {}
+  operating_hours = {}
+  events = {}
   menus = {}
   items = {}
 
+  @staticmethod
+  def update_data(**kwargs):
+    Data.eateries = kwargs.get('eateries')
+    Data.operating_hours = kwargs.get('operating_hours')
+    Data.events = kwargs.get('events')
+    Data.menus = kwargs.get('menus')
+    Data.items = kwargs.get('items')
+
+
+class CoordinatesType(ObjectType):
+    latitude = Int(required=True)
+    longitude = Int(required=True)
+
+    def __init__(self, **kwargs):
+      self.latitude = kwargs.get('latitude')
+      self.longitude = kwargs.get('longitude')
+
+
+class CampusAreaType(ObjectType):
+    description = String(required=True)
+    description_short = String(required=True)
+
+    def __init__(self, **kwargs):
```

Comment:
luckily the graphene `ObjectType` constructor automatically does exactly what all these constructors do so you can leave them out